lib/mini_twitter_client.rb

```diff
@@ -0,0 +1,41 @@
+require 'faraday'
+require 'json'
+
+def connection
+  Faraday.new(
+    url: 'http://localhost:4000/api/tweets',
+    params: {},
+    headers: {
+      'Accept' => 'application/json',
+      'Content-Type' => 'application/json'
+    }
+  )
+end
+
+def tweets
```

Comment:
Personally, I would call it this way

```suggestion
def fetch_tweets
```

or this way

```suggestion
def get_tweets
```

to highlight that it makes a HTTP call under the hood and it is not just a variable with `tweets` that is in your local memory, etc. I think the users of this method should remember that it makes a call to external application. It is important because such remote call may take more time in case of network issues, etc.

Reply:
In beginning it was get_tweets, but rubocop suggest not to use get in the begin of name so I was traying to find correct word, at the end I left tweets, of course it stung my eyes ... the best looks get_tweets

Reply:
anyway, will use fetch_tweets

Reply:
ah, ok. It's https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/AccessorMethodName rule.

It makes sense in typical case, for example:

```ruby
def tweets # do not use get_tweets
  @tweets
end
```

or similar cases.

I guess, personally, I disagree with this Rubocop warning in case of HTTP client ;-)

If you want to, you can disable this rule in your project this way:

```yml
# .rubocop.yml
Naming/AccessorMethodName:
  Enabled: false
```

Reply:
I leave warnings working, but I will decide next time abut agree or not with Rubocop